### PR TITLE
Feat mcp 72 cron job

### PR DESCRIPTION
**Pull request by dgavrischuk**

### CODE REVIEW

File: mcp/kubernetes_locals.tf [hunk 1]
```diff
@@ -42,13 +42,13 @@ locals {
     if ! contains(split("/", kube_file), "terraform")
```
Comment (gbmeuk) [inline mcp/kubernetes_locals.tf:42]: This was an old idea when we thought we were looking for files per micro service but really we should be explicit. Line 38 should just look for the file in the root directory. See https://github.com/mesoform/terraform-infrastructure-modules/blob/master/mcp/locals.tf#L16

**Review by gbmeuk — DISMISSED** [review]

**Review by gbmeuk — CHANGES_REQUESTED** [review]
clearer variable naming. `key` and `value` are ambiguous

File: mcp/kubernetes_locals.tf (hunk 1 above)
Comment (dgavrischuk) [reply mcp/kubernetes_locals.tf:42]: Does this apply only to k8s_secret.yml, or to all yml configuration description files?

Comment (gbmeuk) [reply mcp/kubernetes_locals.tf:41]: All files

**Review by AlexBMes — APPROVED** [review]

**Pull request merged**